pages/agent/v3/elastic_ci_aws.md.erb

```diff
@@ -54,12 +58,10 @@ The Buildkite Elastic CI Stack supports:
 - CloudWatch logs for system and Buildkite agent events
 - CloudWatch metrics from the Buildkite API
 - Support for stable, beta or edge Buildkite Agent releases
-- Create as many instances of the stack as you need
+- Multiple deployments to the same AWS Account
```

Comment:
deployments is an pretty overloaded term here. What about `Run multiple Elastic Stacks on the same AWS account`, not great. `Run Multiple instances on the same AWS account`? meh

I don't have a great answer here.